manager/state/raft/raft.go

```diff
@@ -361,7 +361,7 @@ func (n *Node) JoinAndStart(ctx context.Context) (err error) {
 		if err != nil {
 			n.stopMu.Lock()
 			// to shutdown transport
-			close(n.stopped)
+			n.cancelFunc()
```

Comment:
I'm not very familiar with this part of the code, but I didn't follow why this change is needed.